.gitignore

```diff
@@ -5,7 +5,7 @@
 .mtj.tmp/
 
 # Package Files #
-*.jar
+#*.jar
```

Comment:
I understand Professor @chrismattmann but I don't understand what would be the location of the jar? Are you proposing we would not have the jar in the project repo, even until it is published on the Central repo?